### PR TITLE
[FrameworkBundle] use Table instead of the deprecated TableHelper

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by saro0h** [comment]
You have to fix tests.

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php [hunk 1]
```diff
@@ -72,6 +72,16 @@ public function describe(OutputInterface $output, $object, array $options = arra
     }
 
     /**
+     * Returns the output.
+     *
+     * @return OutputInterface The output
+     */
+    protected function getOutput()
```
Comment (saro0h) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php]: I think that changing the visibility of ``$output`` is enough.

Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php]: Changing the visibility makes it mutable. The methods allows us to keep the property read-only.

**Comment by xabbuh** [comment]
@saro0h Thanks for the hint. I updated the dependencies to make the tests pass again.

**Comment by Tobion** [comment]
:+1:

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Comment by fabpot** [comment]
@xabbuh Can you make a PR for 2.6? Thanks.

**Comment by xabbuh** [comment]
@fabpot Of course, can you please merge the `2.5` branch into the `2.6` branch?

**Pull request merged**